libsrc/apple2/ser/a2.gs.s

```diff
@@ -314,20 +314,19 @@ SER_CLOSE:
 
 SER_OPEN:
         bit     $C082                   ; Check if this is a IIgs
```

Comment:
I'd say, this comment is better placed on its own above this line as it refers rather to the section than this instruction.